src/Load_and_save.jl

```diff
@@ -135,23 +207,23 @@ function load_image(path::String)::Array{MedImage}
-      "xyzt_units", "cal_max", "cal_min", "slice_duration", "toffset", "descrip", "aux_file", "qform_code", "qform_code_name", "sform_code", "sform_code_name", "quatern_b", "quatern_c", "quatern_d",
+      "xyzt_units", "cal_max", "cal_min", "slice_duration", "toffset", "descrip", "aux_file", "qform_code", "qform_code_name", "qfac", "qform_qto_xyz", "sform_code", "sform_code_name", "quatern_b", "quatern_c", "quatern_d",
       "qoffset_x", "qoffset_y", "qoffset_z", "srow_x", "srow_y", "srow_z", "sto_xyz_matrix", "intent_name"]
     metadata_values = ["one", nifti_image_header.dim_info, nifti_image_header.dim, (nifti_image_header.intent_p1, nifti_image_header.intent_p2, nifti_image_header.intent_p3), nifti_image_header.intent_code,
       nifti_image_header.datatype, nifti_image_header.bitpix, nifti_image_header.slice_start, nifti_image_header.pixdim, nifti_image_header.vox_offset, nifti_image_header.scl_slope,
       nifti_image_header.scl_inter, nifti_image_header.slice_end, nifti_image_header.slice_code, nifti_image_header.xyzt_units, nifti_image_header.cal_max, nifti_image_header.cal_min,
       nifti_image_header.slice_duration, nifti_image_header.toffset, (nifti_image_header.descrip, formulate_string(nifti_image_header.descrip)), (nifti_image_header.aux_file, formulate_string(nifti_image_header.aux_file)),
-      nifti_image_header.qform_code, formulate_xform_string(nifti_image_header.qform_code), nifti_image_header.sform_code, formulate_xform_string(nifti_image_header.sform_code),
+      nifti_image_header.qform_code, formulate_xform_string(nifti_image_header.qform_code), calculate_qfac(nifti_image_header.pixdim),
+      formulate_qform_matrix(nifti_image_header.quatern_b, nifti_image_header.quatern_c, nifti_image_header.quatern_d, nifti_image_header.qoffset_x, nifti_image_header.qoffset_y, nifti_image_header.qoffset_z, nifti_image_header.pixdim[2], nifti_image_header.pixdim[3], nifti_image_header.pixdim[4], calculate_qfac(nifti_image_header.pixdim)),
+      nifti_image_header.sform_code, formulate_xform_string(nifti_image_header.sform_code),
       nifti_image_header.quatern_b, nifti_image_header.quatern_c, nifti_image_header.quatern_d, nifti_image_header.qoffset_x, nifti_image_header.qoffset_y, nifti_image_header.qoffset_z,
       nifti_image_header.srow_x, nifti_image_header.srow_y, nifti_image_header.srow_z, formulate_sto_xyz((nifti_image_header.srow_x, nifti_image_header.srow_y, nifti_image_header.srow_z)), nifti_image_header.intent_name]
 
     metadata = Dictionaries.Dictionary(metadata_keys, metadata_values)
 
 
 
-
-
-
+    return [MedImage([voxel_data, spatial_metadata, image_type, image_subtype, voxel_datatype])]
     return [MedImage([nifti_image.raw, nifti_image_header.pixdim[2:4], (nifti_image_header.srow_x[1:3], nifti_image_header.srow_y[1:3], nifti_image_header.srow_z[1:3]), (nifti_image_header.qoffset_x, nifti_image_header.qoffset_y, nifti_image_header.qoffset_z), " ", " "])]
```

Comment:
This line needs to be commented, apprently i missed it for some reason.

Reply:
yes not needed anymore